lib/run.js

```diff
@@ -90,6 +90,17 @@ function findMatch(config, name, version) {
   }
 }
 
+function formatNoProxy(options, browser) {
+  var value = options.noProxy || [];
+  if (typeof value !== 'string') {
+    if (browser === 'chrome') {
```

Comment:
It might be simpler/easier to follow if we had `formatNoProxyFirefox(...)` and `formatNoProxyChrome(...)` functions